script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -141,27 +141,27 @@ As soon as the tension was broken, and my mind was at ease, the headache invades
 [sha:984fc50db689f28fd2215bbcfa246cfc6a134d96]{
 -- Page 372, Offset 36365.
 -- 先輩は右肩に回りこんで俺に肩を貸してくれた。
-Senpai goes around my right shoulder and supports me up from there.
+Senpai goes over to my right shoulder and props me up from there.
 }
 [sha:6ab31a7c4698f9e1e7801201f654def87b9b4afb]{
 -- Page 373, Offset 36366.
 -- 「……ごめんなさい。抱えてあげたいのは山々なんですけど、わたしも荷物があって。……ああもう、こんな時にセブンは起きもしないんですから……！」
-"...I'm sorry. I would love to carry you right now, but I have a lot of luggage as well... Oh come on! It's all because Seven won't get up around this time...!"
+"...I'm sorry. I would love to carry you right now, but I have a lot of luggage as well... Oh come on! This is all because Seven refuses to get up...!"
 }
 [sha:21dabf797460d231ed9386d193aaf0363a8dc255]{
 -- Page 374, Offset 36367.
 -- ……荷物、というのはあの銃の事だろう。
-...Luggage. In other words, that gun of hers.
+...Luggage. In other words, that weapon of hers.
 }
 [sha:4a3493f0e41d9333f57f70ff7c71e37f0ad0eee5]{
 -- Page 374, Offset 36368.
 -- 先輩は片手であの銃を持って、片手で俺を支えて瓦礫を昇り始めた。
-With the gun on one side, and me on the other, Senpai begins to climb up the rubble.
+With her weapon on one side, and me on the other, Senpai begins to climb up the rubble.
 }
 [sha:3e6d5c1ea2d972f9b1d9ac034a9d1fc3d90adbb9]{
 -- Page 375, Offset 36369.
 -- ……いっそ気を失ってしまいたいが、いま眠る訳にはいかない。ここで眠れば次に目覚められる保証はないし、なにより、俺はまだ先輩に伝えたい事が残っている。
-...I'd love to pass out, but I can't allow myself to sleep right now. Because if I do, there's no guarantee I'll wake up again. Not only that, but there's still something I need to tell Senpai.
+...I wish I could pass out right now, but I can't allow myself to fall asleep. If I do, there's no guarantee that I'll wake up again. Not only that, but I still have something that I need to tell Senpai.
 }
 [sha:ca7608c4220b52702fd8614b6bad92f3cc5404f0]{
 -- Page 375, Offset 36370.
```

Comment:
36371: And thus, the curtain falls on Roa's reincarnate.
otherwise the next line doesn't make a damn bit of sense